pxemgr/pxemanager.go

```diff
@@ -212,6 +214,9 @@ func (mgr *pxeManagerT) startIPXEserver() error {
 	mgr.router.Methods("PUT").PathPrefix("/admin/host/{serial}/set_etcd_cluster_token").HandlerFunc(withSerialParam(mgr.setEtcdClusterToken))
 	mgr.router.Methods("PUT").PathPrefix("/admin/host/{serial}/override").HandlerFunc(withSerialParam(mgr.override))
 
+	mgr.router.Methods("GET").PathPrefix("/admin/mayu_config").HandlerFunc(mgr.getMayuConfig)
+	mgr.router.Methods("PUT").PathPrefix("/admin/mayu_config").HandlerFunc(mgr.setMayuConfig)
```

Comment:
Is `PUT /admin/mayu_config` the operation used to create what can later be fetched via `GET /admin/mayu_config`? Then it's alright.

If there is anything else you want my comment on, please give me some detail. I'm lacking some context to otherwise.